### DISCUSSION TITLE
Angle measure hard to pinpoint

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#2974] [Ideas]
Angle measure is an excellent tool, but it currently takes user to click on the starting point and drag the cursor to the end point to get a measurement. That is often a very inaccurate way since it does not select objects, only screen pixels.

Maybe it could work better by allowing user to click on starting and end objects to get a measure, or at least by allowing to click on starting and end pixels instead of dragging a line. Future developments may also include the possibility to define by keyboard lenght and angle. What do you think? Could be useful, feasible and easy to code?

**Comment by alex-w** [comment]
Please check settings of the plugin

**Comment by PascualSalvador** [comment]
I've been trying several times to measure angles by picking starting and end objects but I cannot understand how to set the first object for a second measurement. The tool always keep the same first object for future measurements.
Also, why angle measurement depends on the ecuatorial grid and not on the ecliptical grid? Shouldn't it be grid independant?
